pkg/controller/nodes/executor.go

```diff
@@ -32,12 +32,13 @@ import (
 )
 
 type nodeMetrics struct {
-	Scope              promutils.Scope
-	FailureDuration    labeled.StopWatch
-	SuccessDuration    labeled.StopWatch
-	ResolutionFailure  labeled.Counter
-	InputsWriteFailure labeled.Counter
-	TimedOutFailure    labeled.Counter
+	Scope                   promutils.Scope
+	FailureDuration         labeled.StopWatch
+	SuccessDuration         labeled.StopWatch
+	ResolutionFailure       labeled.Counter
+	InputsWriteFailure      labeled.Counter
+	TimedOutFailure         labeled.Counter
+	InterruptedThresholdHit labeled.Counter
```

Comment:
Should we add a metric for just all Interrupted events? even if they don't hit the threshold I mean...

Reply:
Unfortunately, we are counting system errors towards interrupted threshold. 

However, we are capturing interrupted metrics from kube-state-metrics (not realtime though, infrasepnd model): pod_label_info X pod_info X node_info.

Reply:
I will add a counter for system failures.

Reply:
added a counter